src/app/sommerjobb/page.tsx

```diff
@@ -74,7 +74,7 @@ export default async function Page({
         postcodes = [];
     }
 
-    const sommerjobbKeyword = "sommerjobb";
+    const sommerjobbKeyword = "Utendørs";
```

Comment:
whops, denne var bare for testing, men det er her man sender inn "kategori keywords"

Reply:
ja tenker det skal gå, men bare husk alle søkeordene skal være i samme array så hvis bruker huker av Utendørs og Kultur så har du fortsatt bare en array (searchKeywords) som innholder alle relevante søkeord fra begge kategorier